spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -104,4 +104,66 @@
       end
     end
   end
+
+  describe 'POST #create' do
+    let(:note_params) { { title: 'New Note', note_type: 'review', content: 'Sample content' } }
+
+    context 'when user is logged in' do
+      include_context 'with authenticated user'
+
+      context 'when parameters are valid' do
+        before { post :create, params: { note: note_params } }
+
+        it 'creates a new note' do
+          expect(Note.count).to eq(1)
+        end
+
+        it 'responds with a success message' do
+          expect(response_body['message']).to eq(I18n.t('controller.note_create_success'))
+        end
+
+        it 'responds with 201 status' do
+          expect(response).to have_http_status(:created)
+        end
+      end
+
+      context 'when note_type is invalid' do
+        before { post :create, params: { note: note_params.merge(note_type: 'invalid_type') } }
+
+        it 'does not create a note' do
+          expect(Note.count).to eq(0)
+        end
```

Comment:
Acá también, esta perfecto el enfoque, y si quisieramos generalizarlo aún mas sería algo similar:
```suggestion
        it 'does not create a note' do
           expect { post :create, params: { note: note_params } } .not_to change(Note, :count)
        end
```

Reply:
Acordemonos de corregir esto que decia Meg!